### DISCUSSION TITLE
Curso fullstack .NET

### DISCUSSION THREAD

**Discussion by RafaelDifini** [balta-io/forum#612] [Tira Dúvidas] [answered]
Boa noite, Balta, trabalho em um projeto onde  usamos a arquitetura MVC, um colega  alega ter tido problemas ao fazer múltiplas requisições 
a um mesmo controller com erro de ter mais de um contexto aberto, inicializando ele no próprio construtor da controller. Aí te pergunto, isso não vai ocorrer se mais de um usuário fizer uma requisição simultanea a um mesmo handler? Se não, porque? E quando digo requisição vamos imaginar também um caso que sejam 2 usuários diferentes dando update em um mesmo objeto

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @RafaelDifini, como vai?

Qual tipo de problema ele teve? Preciso de uma descrição mais detalhada para poder te orientar.
